### PR TITLE
fix #7713 feat(nimbus): Add targeting for first run

### LINKED ISSUE #7713: Add Mobile First Run Targeting
Let's add a new targeting config for mobile first run that uses the boolean property that Fenix and Firefox iOS expose that is true on the first session, and then set both sticky and first run options on the targeting config to true.

Blocked on https://github.com/mozilla/experimenter/issues/7712

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2775)

### PR DESCRIPTION
**Pull request by yashika-khurana**
Because

* We want to add a new targeting config for mobile first run that uses the boolean property that Fenix and Firefox iOS expose that is true on the first session, and then set both sticky and first run options on the targeting config to true.

This commit

* Adds a new first run targeting for mobile
 fixes #7713

### CODE REVIEW

**Comment by jaredlockhart** [comment]
Oh no, I think there's another property, called something like `isFirstRun`, @jhugman do you know?  I think it's only available on one of the mobile apps and not another?

**Review by jaredlockhart — DISMISSED** [review]
Oh no, I think there's another property, called something like `isFirstRun`, @jhugman do you know? I think it's only available on one of the mobile apps and not another?

**Comment by eliserichards** [comment]
we have [this](https://github.com/mozilla-mobile/fenix/blob/09b11da8c2f8545484258bc05c122e6a335bf67a/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#L390) on Fenix. I'm not sure how to do it on iOS 🤔

**Comment by jhugman** [comment]
- [`isFirstRun` in Fenix][1]
- [`isFirstRun` in Firefox for iOS][2]

[1]: https://github.com/mozilla-mobile/fenix/blob/911f3506420a2accbd7114e16783a0fd93edc0c2/app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt#L93
[2]: https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Application/AppLaunchUtil.swift#L151

**Comment by jaredlockhart** [comment]
Thanks @jhugman !  @yashika-khurana Can you update this to use that expression for the relevant apps pls 🙏

File: app/experimenter/targeting/constants.py [hunk 1]
```diff
@@ -177,6 +177,22 @@ def __post_init__(self):
...
     ),
 )
+MOBILE_FIRST_RUN_USER = NimbusTargetingConfig(
+    name="First run Users on Mobile",
+    slug="mobile_first_run",
+    description=(
+        "First-run users on Fenix and Firefox for iOS \
+            who installed the app less than 3 days ago"
+    ),
+    targeting="days_since_install < 3",
```
Comment (jhugman) [inline app/experimenter/targeting/constants.py:187]:
```suggestion
    targeting="isFirstRun == 'true'",
```

**Review by jhugman — APPROVED** [review]
r+ with comments; mainly to get this out of my review queue.

**Comment by yashika-khurana** [comment]
checks are not happy, let me fixed that

**Comment by yashika-khurana** [comment]
@jaredlockhart @jhugman thank you for the feedback 🎉

**Pull request merged**